### PR TITLE
API-3758: Add test for MarkdownPage

### PR DESCRIPTION
**Pull request by anthony-kopczyk**
### Description
Added a unit test for the MarkdownPage component.

Jira Ticket: https://vajira.max.gov/browse/API-3758

### Process
- [x] Tests added or updated for change
- [N/A] Docs updated
  - [N/A] If you have made structural changes to the site, the [Developer Portal Content Types](https://community.max.gov/x/qUwOg) docs have been updated
  - [N/A] If you have made/confirmed any decisions about user interactions and accessibility, the [Accessible Component Design](https://community.max.gov/x/xS8mgg) docs have been updated
- [N/A] Accessibility concerns have been considered and addressed
  
  - [N/A] If you've made any UI changes, you've run an axe check using the axe browser extension that reported no new issues

### Requested feedback
I'm wondering if this page should have a role assigned on the section tag. I'm not sure what that role would be though. If this is something we want to do, let me know and I can do it under this ticket. Maybe it would make sense to use article instead of section? Let me know what you think!

### CODE REVIEW

**Review by johanna-adhoc — DISMISSED** [review]
LGTM

File: src/components/markdownPage/MarkdownPage.test.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+const markdownComponent: MarkdownComponent = (): JSX.Element => (
+  <div data-testid={markdownComponentTestId}>I am a markdown component</div>
+);
+
+describe('Markdown Page', () => {
+  it('should render the given markdown', () => {
+    // The markdown is a react component, so we pass a regular/non-markdown component to the MarkdownPage for testing
+    render(MarkdownPage(markdownComponent));
+    const markdownComp = screen.getByTestId(markdownComponentTestId);
+    expect(markdownComp).toBeDefined();
```
[the remark this answers is not in the document]
Comment (anthony-kopczyk) [reply src/components/markdownPage/MarkdownPage.test.tsx:17]: Yeah. That's the main thing I was unsure of while working this. I'm passing a child component and checking it's there using the test id. I could use getByText instead to get 'I am a markdown component'.

Oher than that, this component is just a wrapper including some padding in the form of css classes. It also adds a `section` html tag. I could see testing for the section tag, but I didn't find any built-in accessibility type things for section - there's no text, so I'd need to use getByRole or something else. I was wondering if I should change the tag to article or add an aria role - then I could check that it adds the article wrapper properly. But yeah, this is basically a section tag, that renders the given child component, and that's all it does..

[the remark this answers is not in the document]
Comment (johanna-adhoc) [reply src/components/markdownPage/MarkdownPage.test.tsx:17]: oh I would check for the classnames vads-u-padding-y--5 , vads-l-grid-container

[the remark this answers is not in the document]
Comment (anthony-kopczyk) [reply src/components/markdownPage/MarkdownPage.test.tsx:17]: I'm about to start on this, but had a thought last evening. Should we be checking for the css classes in these tests?  The visual regression tests validate that none of the visual page look has changed. Are the visual regression tests alone sufficient in this regard?

[the remark this answers is not in the document]
Comment (johanna-adhoc) [reply src/components/markdownPage/MarkdownPage.test.tsx:17]: I am actually not sure... if the visual regression test only checks pixel matching then I would say no that they don't do the same. But then again when we say that we are creating a unit test for this component it should cover all that the component is, and in this particular case the styling that is applying.
That is one of the reasons that having all theses separate lines of testing is confusing for me, how do you keep track what was done with one and not with the other... how you you assets the current coverage for a component when all of it is spread

[the remark this answers is not in the document]
Comment (anthony-kopczyk) [reply src/components/markdownPage/MarkdownPage.test.tsx:17]: Sounds good. I'm going to look into the styling test.

[the remark this answers is not in the document]
Comment (anthony-kopczyk) [reply src/components/markdownPage/MarkdownPage.test.tsx:18]: The styling test should be added. I used the standard DOM api, so I didn't need to add test ids. I can switch this over if needed.

**Review by johanna-adhoc — APPROVED** [review]
LGTM

**Review by johanna-adhoc — APPROVED** [review]

**Review by TS000 — APPROVED** [review]
LGTM ✨

**Pull request merged**